request_test.go

```diff
@@ -2,6 +2,7 @@ package sarama
 
 import (
 	"bytes"
+	"reflect"
```

Comment:
Hmm, what harm in that since it is a standard library. Besides it is only used in tests to perform deep comparisons?